src/main/scala/sbtlicensereport/SbtLicenseReport.scala

```diff
@@ -36,6 +36,7 @@ object SbtLicenseReport extends AutoPlugin {
     val dumpLicenseReportAnyProject = taskKey[File](
       "Dumps a report file against all projects of the license report (using the target language) and combines it into a single file."
     )
+    val checkLicenses = taskKey[Unit]("Checks that all licenses are allowed. Fails if other licenses are found.")
```

Comment:
`updateLicenses` flaunts it, but generally I think we should try to name tasks with "plugin-specific-prefix + something", so I think `licenseCheck` is better, like `scalafmtCheck`